docs/source/common/lightning_module.rst

```diff
@@ -705,21 +705,7 @@ Tasks can be arbitrarily complex such as implementing GAN training, self-supervi
 When used like this, the model can be separated from the Task and thus used in production without needing to keep it in
 a ``LightningModule``.
 
-- You can export to `ONNX <https://pytorch.org/docs/stable/onnx.html>`_ using :meth:`~pytorch_lightning.core.lightning.LightningModule.to_onnx`.
-- Or trace using `TorchScript <https://pytorch.org/docs/stable/jit.html>`_ using :meth:`~pytorch_lightning.core.lightning.LightningModule.to_torchscript`.
```

Comment:
I believe one of the examples should stay here. It's not about fully de-duplicating everything in the docs. We can show a small example like below here and point to the more extensive docs via link you made. I believe that would be appropriate.